types/xrm/index.d.ts

```diff
@@ -4223,6 +4245,10 @@ declare namespace Xrm {
              *  The message to be displyed in the alert dialog.
              */
             text: string;
+            /**
+             * (Optional) String. The title of the alert dialog.
```

Comment:
```suggestion
             * The title of the alert dialog.
```
(the TypeScript providees the rest.
If not, JSDoc would be better:
```ts
             * @prop {string} [title] The title of the alert dialog.
```